includes/abstract.tex

```diff
@@ -1,10 +1,10 @@
 \begin{abstract}
     In this paper, we introduce \fuzzer{}: a modular \smt{} problem instance
-    transformer and generator for string solvers. We supply a
```

Comment:
The "for string solvers" part was initially conceived by Vijay, and I think it's worth it to keep it. Do you feel it's better without it?